### PR TITLE
[vsphere] add workflow for multi zone cluster

### PR DESCRIPTION
**Pull request by jinyunma**
[SPLAT-773](https://issues.redhat.com/browse/SPLAT-773) add workflow for multi zone cluster

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/provision/cucushift-installer-rehearse-vsphere-ipi-zones-provision-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-zones-provision
+  steps:
+  - chain: ipi-conf-vsphere-zones
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/provision/cucushift-installer-rehearse-vsphere-ipi-zones-provision-chain.yaml:4]: From `ipi-conf-vsphere-zones.ref`, I saw it enables `FeatureGate`, so vsphere multip-zones is a TP feature?

Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/provision/cucushift-installer-rehearse-vsphere-ipi-zones-provision-chain.yaml:4]: yes, it is TP in 4.12.

Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/provision/cucushift-installer-rehearse-vsphere-ipi-zones-provision-chain.yaml:4]: If we don't enable featureGate in install-config.yaml with zone configuration, will get following error.
```
time="2022-10-08T04:00:19Z" level=error msg="failed to fetch Metadata: failed to load asset \"Install Config\": failed to create install config: invalid \"install-config.yaml\" file: [platform.vsphere.failureDomains: Forbidden: the TechPreviewNoUpgrade feature set must be enabled to use this field, platform.vsphere.vcenters: Forbidden: the TechPreviewNoUpgrade feature set must be enabled to use this field]"
```

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/provision/cucushift-installer-rehearse-vsphere-ipi-zones-provision-chain.yaml:4]: Thanks for the info. If that, we can follow other techpreview workflows rules, place the steps in `cucushift/installer/rehearse/vsphere/ipi/zones/techpreview`, and document the workflow get supported from 4.12.

**Pull request merged**